src/components/chat/ChatListRow.tsx

```diff
@@ -0,0 +1,150 @@
+import styled from 'styled-components';
+
+import { ChatRoomSummary } from '../../types';
+
+import transformDate from '../../utils/transformDate';
+
+interface ChatListRowProps {
+  chatRoomSummary : ChatRoomSummary;
+}
+
+function ChatListRow({ chatRoomSummary }: ChatListRowProps) {
+  const displayUnreadCount = (count: number) => {
+    if (count > 999) return <span>+999</span>;
+    if (count > 0) return <span>{count}</span>;
+    return null;
+  };
+
+  return (
+    <ChatListItem>
+      <div>
+        <img
+          src={chatRoomSummary.receiverImageUrl}
+          alt={chatRoomSummary.receiverName}
+        />
+      </div>
+      <div>
+        <div>
+          <b>{chatRoomSummary.receiverName}</b>
+          <p>{chatRoomSummary.lastMessage}</p>
+        </div>
+        <div>
+          <small>{transformDate(chatRoomSummary.lastMessageDate)}</small>
+          {displayUnreadCount(chatRoomSummary.unreadMessageCount)}
+        </div>
+      </div>
+    </ChatListItem>
+  );
+}
```

Comment:
p4: 이렇게 작성하시는건 어떤가요?

```suggestion
function ChatListRow({ chatRoomSummary }: ChatListRowProps) {
  const {receiverName, receiverImageUrl, lastMessage, unreadMessageCount, lastMessageDate } = chatRoomSummary;

  const displayUnreadCount = (count: number) => {
    if (count > 999) return <span>+999</span>;
    if (count > 0) return <span>{count}</span>;
    return null;
  };
  
  
  return (
    <ChatListItem>
      <div>
        <img
          src={receiverImageUrl}
          alt={receiverName}
        />
      </div>
      <div>
        <div>
          <b>{receiverName}</b>
          <p>{lastMessage}</p>
        </div>
        <div>
          <small>{transformDate(lastMessageDate)}</small>
          {displayUnreadCount(unreadMessageCount)}
        </div>
      </div>
    </ChatListItem>
  );
}
```